apis/v1alpha1/backendpolicy_types.go

```diff
@@ -55,33 +64,43 @@ type BackendPolicySpec struct {
 	// the oldest BackendPolicy.
 	//
 	// Support: Core
+	//
+	// +kubebuilder:validation:MinItems=1
```

Comment:
I'm not sure we want this change. I think we want it to be possible to specify a BackendPolicy without applying it to any specific backends. This could be helpful if you want to provide a standard set of policies that can be applied to different backends, even if not all of them apply to a backend at once.

Reply:
@robscott if that's the case, then shouldn't this field be optional? I do like your idea for a nil backendRefs, but the docs and field requirements indicate otherwise.

Reply:
This is similar to my comment on the other issue, but in some cases I think it's helpful to actually show an empty list instead of just omitting the field entirely. I'm curious what the behavior is now, I'm assuming a user can create a BackendPolicy with `[]` for this field, do they even need to specify that? I don't read anything in the docs that specifically requires a BackendRef to be set, but we have not made it clear enough that empty is a valid option here, that would definitely be good to update.

Reply:
> I think it's helpful to actually show an empty list instead of just omitting the field entirely.

+1. I'm not versed in conventions but from a user's perspective, this is very helpful for some fields.

Reply:
> I'm assuming a user can create a BackendPolicy with [] for this field, do they even need to specify that?

Other than `port`, all  `BackendRef ` fields are required so an empty `BackendRefs` is not permitted.